modules/display/display_rate_interpolation.py

```diff
@@ -0,0 +1,49 @@
+import sys
+sys.path.append('../..')
+sys.path.append('../database/')
+sys.path.append('../display/')
+sys.path.append('../update/')
```

Comment:
to chyba jest zbędne, nie testujemy już chyba funkcji tutaj. Użyj jej w notebooku dla testów ;)